### PR TITLE
android build config

### PR DESCRIPTION
**Pull request by PhongTaKiai**
- close: #42

### CODE REVIEW

**Comment by mafreud** [comment]
@PhongTaKiai Please share jks and key properties with us on Slack.

**Comment by rick1919** [comment]
@PhongTaKiai 
Thank you for the invitation.

I've finished installing the file Android/iOS, so please approve it!

File: android/app/build.gradle [hunk 1]
```diff
@@ -22,6 +22,12 @@ if (flutterVersionName == null) {
...
 }
 
+def keystoreProperties = new Properties()
+def keystorePropertiesFile = rootProject.file('key.properties')
+if (keystorePropertiesFile.exists()) {
+    keystoreProperties.load(new FileInputStream(keystorePropertiesFile))
+}
+
 android {
     namespace "com.example.app_buy_sell"
```
Comment (mafreud) [inline android/app/build.gradle:32]: As far as I know, Google doesn't allow us to use "example" in bundle id. Could you change it to com.app_buy_sell.release?

**Review by mafreud — CHANGES_REQUESTED** [review]

**Comment by PhongTaKiai** [comment]
@mafreud i updated namespace same as app ID

**Comment by mafreud** [comment]
@PhongTaKiai as I mentioned, could you change it to ```com.app_buy_sell.release```?

**Comment by PhongTaKiai** [comment]
> @PhongTaKiai as I mentioned, could you change it to `com.app_buy_sell.release`?

ok, i updated to com.app_buy_sell.release

**Comment by mafreud** [comment]
@PhongTaKiai Thanks. Please update the Firebase config so that the app can connect to Firebase.

**Comment by PhongTaKiai** [comment]
> @PhongTaKiai Thanks. Please update the Firebase config so that the app can connect to Firebase.

@mafreud Can you explain more, change namespace or appId?

**Comment by mafreud** [comment]
@PhongTaKiai Since the app id has been changed, you need to update google-service.json to connect to Firebase.

**Comment by PhongTaKiai** [comment]
> @PhongTaKiai Since the app id has been changed, you need to update google-service.json to connect to Firebase.

@mafreud namespace and appid are different, 
now appid is com.ailab.appBuySell, do you want change appid?

**Comment by mafreud** [comment]
@PhongTaKiai If your build can connect to the Firestore in your environment, it's okay.

**Comment by PhongTaKiai** [comment]
@mafreud Using firebase is no problem, I have updated the namespace, please review again

**Comment by mafreud** [comment]
Please test the build and upload the video. @PhongTaKiai

**Comment by PhongTaKiai** [comment]
https://github.com/ailab-dev/app_buy_sell/assets/117698659/ee8d42cf-8d69-41e9-b32c-cf1654e0faf4

@mafreud uploaded video

**Review by mafreud — APPROVED** [review]
LGTM 🎉

**Pull request merged**